src/exposed.tsx

```diff
@@ -49,6 +51,7 @@ export const FormuleContext = ({
               customFields,
               customWidgets,
               transformSchema,
+              hideAnchors,
```

Comment:
why do we need it here?

Reply:
For users to be able to decide if they want to disable anchors